src/common/HPM.c

```diff
@@ -322,13 +326,13 @@ void hplugins_removeFromHPData(enum HPluginDataTypes type, uint32 pluginID, stru
 	struct hplugin_data_entry *entry;
 	int i;
 
+	if (!store)
```

Comment:
Same as the above case, `store` can be NULL, and it gets initialized in the function (it's marked as [in,out] in the documentation)